### ISSUE TITLE
Territory letters difficult to read when background is dark

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1766]
**Describe the bug**
Territory letters difficult to read when background is dark. This is an a11y issue

**To Reproduce**
1. Go to any national map where territories have a very high quantile coloring

**Expected behavior**
Letters atop territory circles should be readable

**Screenshots**
![Screen Shot 2022-08-25 at 11 57 46 AM](https://user-images.githubusercontent.com/41567007/186736577-7d63a4e9-4bd6-47ca-a676-cbd18d17846e.png)

**Additional context**
Vega is extremely difficult to work with for this; we should consider using real HTML label below the simple colored circles generated as Choropleth maps (just eliminate the Vega labeling)
